pom.xml

```diff
@@ -87,6 +87,12 @@
 			<artifactId>assertj-core</artifactId>
 			<version>3.18.1</version>
 		</dependency>
+        <!-- https://mvnrepository.com/artifact/commons-validator/commons-validator -->
+        <dependency>
+            <groupId>commons-validator</groupId>
+            <artifactId>commons-validator</artifactId>
+            <version>1.4.0</version>
```

Comment:
Could we use the newer 1.7 version? https://mvnrepository.com/artifact/commons-validator/commons-validator/1.7